tests/test_multilabel_rank.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (C) 2017-2022  Cleanlab Inc.
```

Comment:
Could we rename this test file to: `test_multilabel_classification.py` and try to move all the easily moveable multi-label related tests in here eventually?  @aditya1503 what do you think?

Reply:
I agree